contracts/core/PolicyEngine.sol

```diff
@@ -105,6 +119,10 @@ abstract contract PolicyEngine is IPolicyEngine {
         Operation operation,
         bytes memory context
     ) public view virtual returns (address) {
+        if (_allowedCalls(to, value, data, operation)) {
+            return address(0);
```

Comment:
I thought about using `address(this)` instead, but actually like the 0 address better in the end (since no contract can exist there) and is more explicit than having a policy that is its own engine.

(I.e. I like it like this! Just wanted to share my thought experiment)